lib/src/package_utils.dart

```diff
@@ -33,3 +33,50 @@ String getPackageDescription(String directoryName) {
 
 String getPackageVersion(String directoryName) =>
     _getPubspec(directoryName)['version'];
+
+const String SDK_INTRO = """
```

Comment:
Can we please pull this in via a command-line arg? It doesn't feel right to have SDK-specific verbage in a generic dartdoc tool.